lib/board.ml

```diff
@@ -1,3 +1,14 @@
+open Tile
+open Layout
+(* open LTerm_geom *)
+
 type board = {
-  cursor : int * int;
+  cursor : position;
+  tiles : tile list;
 }
+
+(* let set_tile (b : board) (t : tile) = { cursor = b.cursor; opt = Some
+   t } *)
+
+(* let is_letter (t : tile) = match t with 'a' .. 'z' | 'A' .. 'Z' ->
+   Some t | _ -> None *)
```

Comment:
Are these and the open statement above meant to be commented out or are they supposed to be implemented?

Reply:
these are commented out for now cause I had implemented them before but not sure if might need them later